applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_implicit_dynamic_solver.py

```diff
@@ -67,13 +73,36 @@ def _create_solution_scheme(self):
         elif(scheme_type == "bossak"):
             damp_factor_m = self.dynamic_settings["damp_factor_m"].GetDouble()
             mechanical_scheme = KratosMultiphysics.ResidualBasedBossakDisplacementScheme(damp_factor_m)
+        elif("bdf" in scheme_type or scheme_type == "backward_euler"):
+            if (scheme_type == "backward_euler"):
+                order = 1
+            else:
+                order = int(scheme_type.replace("bdf",""))
+            
+            # Warning
+            if (order > 2):
+                print("WARNING:: BDF Order: ", order, " constant time step must be considered")
+                
+            # In case of rotation dof we declare the dynamic variables
+            if self.settings["rotation_dofs"].GetBool():
+                dynamic_variables = KratosMultiphysics.Parameters("""
+                {
+                    "variable"              : ["DISPLACEMENT","ROTATION"],
```

Comment:
i don't think it is a good idea to provide the variables from the inpu... also how would you inform the element of that?

Reply:
also surely this should NOT be done in the default solver

Reply:
My original idea was to take directly the dof and the each variable toi have the definition of the derivatives. Like currently this is not the case I add the variables as parameters. With this any kind of dynamic problems can be solved (not just one variable at a time, like displacements or velocities)

Reply:
The default solver takes just displacement, this is for rotational cases (this way it is avoided to build a scheme for every variable combination)